data-prepper-core/src/integrationTest/java/com/amazon/dataprepper/integration/EndToEndTestData.java

```diff
@@ -2,7 +2,7 @@
 
 import io.opentelemetry.proto.trace.v1.Span;
 
-public enum ServiceMapTestData {
+public enum EndToEndTestData {
```

Comment:
Maybe even `EndToEndTestSpan` to indicate what functionality this data is expected to provide